templates/extract_bin_file.ipynb

```diff
@@ -1,43 +1,101 @@
 {
```

Comment:
<p>This should get a 3b_prefix, and moved to the main part of the repo. Update the hyper link in the readme as well. </p><p> </p><p>In addition, a 1-2 sentence description of what the notebook does, the making a copy to avoid merge conflicts text, etc. The same stuff all the main notebooks have </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>